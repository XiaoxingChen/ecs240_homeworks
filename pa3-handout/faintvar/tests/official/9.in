c Nested loops
p 4 6 7 1 6
c bb1: v1 = c;
b 1 1
e 1 2
c bb2: ;
b 2
e 2 3
e 2 6
c bb3: v3 = v2;
b 3 3 2
e 3 4
e 3 5
c bb4: print(v4);
b 4 0 4
e 4 3
c bb5: v4 = v2;
b 5 4 2
e 5 2
c bb6: print(v1);
b 6 0 1
