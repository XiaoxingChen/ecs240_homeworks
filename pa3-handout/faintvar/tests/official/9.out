fvin 1 1 3
fvin 2 3
fvin 3 3
fvin 4 3
fvin 5 3 4
fvin 6 2 3 4
